audbackend/core/backend/base.py

```diff
@@ -3,6 +3,11 @@
 import tempfile
 import typing
 
+# Backend type to annotate Backend.create().
+# From Python 3.11 on we can use typing.Self instead.
+# See https://stackoverflow.com/a/71118499
+from typing_extensions import Self
```

Comment:
Since locally I am running Python 3.10 it gives me:

```
ModuleNotFoundError: No module named 'typing_extensions'
```

Reply:
I had to add it as a dependency, but it should be small.